packages/graphql/src/server.js

```diff
@@ -15,7 +15,8 @@ try {
 
 setNetwork(process.env.NETWORK || 'test', {
   performanceMode: false,
-  useMetricsProvider: process.env.USE_METRICS_PROVIDER === 'true'
+  useMetricsProvider: process.env.USE_METRICS_PROVIDER === 'true',
+  proxyAccountsEnabled: process.env.PROXY_ACCOUNTS_ENABLED === 'true'
```

Comment:
I don't see that used anywhere so I assume it is for future ?

Reply:
Right now proxies are enabled with a localStorage var but we need the GraphQL server to always have it on for performance mode

Reply:
Got it. Sg.